pkg/odo/cli/deploy/deploy.go

```diff
@@ -105,28 +106,36 @@ func (o *DeployOptions) Complete(cmdline cmdline.Cmdline, args []string) (err er
 		return err
 	}
 
-	envFileInfo, err := envinfo.NewEnvSpecificInfo(o.contextDir)
+	// ENV.YAML
+	//
+	// Set the appropriate variables in the env.yaml file
+	//
+	// TODO: Eventually refactor with code in dev/dev.go
+
+	envfileinfo, err := envinfo.NewEnvSpecificInfo("")
```

Comment:
Just curious as to why you replaced `o.contextDir` by `""`.. I think it makes sense to leave it explicit here, as it represents the current working directory, no?

Reply:
To prevent using context anymore and since it's the same code for both `dev.go` and `deploy.go`, that's why it's "" :)